### DISCUSSION TITLE
Similar TODO / Expense Tracking app?

### DISCUSSION THREAD

**Discussion by artiekra** [Razeeman/Android-SimpleTimeTracker#240] [General]
Hey everyone! I've been enjoying this time tracker for more then a month already, and now am looking for a good TODO app & expense tracking app..

Asking here, any users of this time tracker - what apps are you using for those purposes? Haven't found those from this dev.. Preferrably, well, something open-source, with comparable amount of features to the time tracker (because tbh, i went through quite a lot of apps, and they are all trying to be extremely "minimalistic", to the point of not even allowing custom categories lol), also, i really like the style of the time tracker..

Anyway, hoped to ask in case someone has found good alternatives, thanks)

**Comment by Razeeman** [comment]
Hello! Maybe Cashew. Looks good. 
https://github.com/jameskokoska/Cashew
